docs/cassandra.adoc

```diff
@@ -26,6 +26,7 @@ image:modes-local.png[]
 Cassandra can be run as a standalone database on the same local host as JanusGraph and the end-user application. In this model, JanusGraph and Cassandra communicate with one another via a `localhost` socket. Running JanusGraph over Cassandra requires the following setup steps:
 
 . http://cassandra.apache.org/download/[Download Cassandra], unpack it, and set filesystem paths in `conf/cassandra.yaml` and `conf/log4j-server.properties`
+. If you want to be able to connect with the Gremlin Console you'll need to enable Cassandra Thrift. To do so open `conf/cassandra.yaml` and update `start_rpc: false` to `start_rpc: true`. If Cassandra is already running Thrift can be started manually with `bin/nodetool enablethrift`. the Thrift status can be verified with `bin/nodetool` statusthrift.
```

Comment:
'If you want to be able to connect with the' -> To?

Reply:
Reworded and pushed to [gh-pages on my fork](https://chupman.github.io/docs.janusgraph.org/latest/cassandra.html#_local_container_mode) to make review easier.